src/main/resources/templates/db_scripts/schema.json

```diff
@@ -652,6 +652,11 @@
           "targetTable": "location",
           "tOps": "ADD"
         },
+        {
+          "fieldName": "effectiveLocationId",
+          "targetTable": "location",
+          "tOps": "ADD"
+        },
```

Comment:
Why has a database index been introduced for this property? Is this a pre-emptive step because this is going to be searchable in future?

Reply:
Since both permanent and temporary location IDs are searchable and have their own indexes, I thought this one might need one as well.  If that's not the case, I'll remove.

Reply:
I think it is reasonable to include it. I only wanted to understand the reasoning for doing so.

Reply:
I'd better add a foreign key instead. it will also create a b-tree index.

Reply:
I think that's what I've done here (isn't it?). This entry is in the foreign keys array.

Reply:
Yes, you did. Sorry again :(